app/controllers/api/v2/providers_controller.rb

```diff
@@ -124,6 +125,16 @@ def update_enrichment
         enrichment.save
       end
 
+      def update_ucas_contacts
+        return if ucas_contact_params.blank?
+
+        ucas_contact_params.keys.each do |type|
+          contact = @provider.contacts.find_or_initialize_by(type: type.gsub(/_contact$/, ''))
```

Comment:
I was wondering about user input on `gsub` so I asked the internet https://security.stackexchange.com/q/216816/158425